ci/pipelines/deploy-to-test.yml

```diff
@@ -975,6 +976,48 @@ jobs:
         params:
           image: ledger-ecr-registry-test/image.tar
           additional_tags: ledger-ecr-registry-test/tag
+
+  - name: ledger-db-migration
+    plan:
+      - get: pay-ci
+      - get: ledger-ecr-registry-test
+        params:
+          format: oci
+        trigger: false
+        passed: [deploy-ledger]
+      - task: assume-role
+        file: pay-ci/ci/tasks/assume-role.yml
+        params:
+          AWS_ROLE_ARN: arn:aws:iam::((pay_aws_test_account_id)):role/concourse
+          AWS_ROLE_SESSION_NAME: terraform-staging-assume-role
```

Comment:
ah thanks no, but we've got that same mistake in every other job in the pipeline... I'll update for this job and correct the others in a separate PR. `AWS_ROLE_SESSION_NAME` is seemingly just an identifier for the role session once its created, its been assuming the correct role in test which is controlled by `AWS_ROLE_ARN` but still we should correct it to avoid confusion ([aws assume role docs](https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html))

Will update, one second.. 👍

Reply:
I think the other jobs are assuming the staging role so they can push the Docker image to the staging ECR repo? We don't have to worry about ECR following a DB migration, do we?

Reply:
hmm they're all using this role ARN, which I'm reading to be the role in test account for concourse to assume, which I presume has the power to push to staging ecr?
>  AWS_ROLE_ARN: arn:aws:iam::((pay_aws_test_account_id)):role/concourse

yeah, there is no ECR activity following a db migration 👍

Reply:
Ahh yes you're right. You'd think I'd be up on my Concourse permissions after spending all day writing about them 😹 OK, cool - if it's just the name that's wrong that makes sense.